sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/Azure.Messaging.EventHubs.Processor.csproj

```diff
@@ -5,6 +5,7 @@
     <PackageTags>Azure;Event Hubs;EventHubs;.NET;Event Processor;EventProcessor;$(PackageCommonTags)</PackageTags>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
     <EnableFxCopAnalyzers>false</EnableFxCopAnalyzers>
+    <RunApiCompat>false</RunApiCompat>
```

Comment:
EventHubs is not yet GA'ed.

Reply:
Packages haven't been released yet, but will shortly (should be this week).   The public API should be stable and we probably want to enforce that.